modules/end-user-guide/pages/mounting-volumes-secrets-and-configmaps.adoc

```diff
@@ -0,0 +1,16 @@
+:navtitle: Mounting Volumes, Secrets and ConfigMaps
+:keywords: user-guide, configuring, user, configmaps, secrets, volumes, mounting, mount
+:page-aliases: 
+
+[id="mounting-volumes-secrets-and-configmaps_{context}"]
+= Mounting Volumes, Secrets and ConfigMaps
+
+Users are able to mount their existing Persistent Volume Claims (PVC), Secrets and ConfigMaps into their {devworkspace} containers. 
```

Comment:
```suggestion
You can mount your existing Persistent Volume Claims (PVC), Secrets and ConfigMaps into your {devworkspace} containers. 
```
BTW, it's not clear to me why I need {devworkspace} containers as a user.

Reply:
@l0rd, I discussed it with David today.

Looking at the GUI and CLI usage, it appears that David's choice of `DevWorkspaces` is the best here.

Now I need to think more about how to present it in the text.

Reply:
Hello, sorry, I think I'm wrong here. In my latest commit I went with just `workspace` or `workspace containers` instead of `DevWorkspace containers` for now, to keep things simple, and to avoid explaining a new term (DevWorkspace) in this page.

Reply:
@dkwon17, in my latest counterPR, I propose writing it as follows:

... the `DevWorkspace` containers [in the cluster] ...